# References

1. Patra, D., Mishra, S., Mohanty, J., Subhadarsani, S., & Sahoo, J. P. (2023). *Plant-Associated Microbial Communities in Agriculture: From Genome Sequencing to Metagenomic Insights*. Academia Molecular Biology and Genomics.

2. Sharuddin, S. S. N., et al. (2020). *Microbial diversity in oil palm (Elaeis guineensis) rhizosphere using metagenomics*. BMC Microbiology, 20(1), 1–13.

3. AfSIS (2021). *Africa Soil Information Service*. [https://africasoils.net](https://africasoils.net)

4. Franzosa, E. A., et al. (2018). *Species-level functional profiling of metagenomes and metatranscriptomes*. Nature Methods, 15(11), 962–968.

5. Douglas, G. M., et al. (2020). *PICRUSt2 for prediction of metagenome functions*. Nature Biotechnology, 38(6), 685–688.

6. Zhang, X., et al. (2022). *Integrative multi-omics reveals microbial pathways influenced by mineral amendments in sustainable agriculture*. Frontiers in Microbiology, 13, 857936.

7. KEGG Pathway Database. [https://www.genome.jp/kegg/](https://www.genome.jp/kegg/)

8. EggNOG-mapper. [http://eggnog-mapper.embl.de](http://eggnog-mapper.embl.de)

9. MG-RAST Metagenomics Analysis Server. [https://www.mg-rast.org](https://www.mg-rast.org)
